### PR TITLE
Bumping ncbi-ngs-sdk to version 2.9.6

### PR DESCRIPTION
**Pull request by r78v10a07**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/ncbi-ngs-sdk/build.sh [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
+if [[ $OSTYPE == darwin* ]]; then
+     export LDFLAGS="${LDFLAGS} -headerpad_max_install_names"
+fi
+
+ln -s $BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-gcc $BUILD_PREFIX/bin/gcc
+ln -s $BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-c++ $BUILD_PREFIX/bin/g++
+ln -s $BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-cc $BUILD_PREFIX/bin/cc
+ln -s $BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-c++ $BUILD_PREFIX/bin/c++
+ln -s $BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-ar $BUILD_PREFIX/bin/ar
+ln -s $BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-ld $BUILD_PREFIX/bin/ld
```
Comment (epruesse) [inline recipes/ncbi-ngs-sdk/build.sh]: This is not future proof. The compilers used by conda-build are exported as $CXX and $CC. I'm surprised this passed on OSX. The name of the compiler there should be different.

Comment (epruesse) [reply recipes/ncbi-ngs-sdk/build.sh]: Technically, `configure` should pick up on the $CC/$CXX settings in the environment.

Often, you can use command line overrides for make to get the right values:

```
make CC=$CC CXX=$CXX
```

**Comment by epruesse** [comment]
Looking at the log, the build does not use the `$CXXFLAGS` etc set by conda-build. It would be good if you could get the build system to use those.

**Comment by dpryan79** [comment]
No longer needed, since I merged in 2.10.0 https://github.com/bioconda/bioconda-recipes/pull/14642

**Pull request closed without merging**